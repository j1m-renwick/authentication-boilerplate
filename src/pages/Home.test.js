import React from 'react';
import { render } from '@testing-library/react';
import Home from "./Home";
import 'jest-styled-components';

test('component renders as expected', () => {
  const component = render(<Home />).baseElement;
  expect(component).toMatchSnapshot();
});

