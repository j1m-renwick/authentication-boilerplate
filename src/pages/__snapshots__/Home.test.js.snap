// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders as expected 1`] = `
.c0 {
  box-sizing: border-box;
  max-width: 50%;
  margin: 0 auto;
  padding: 0 2rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  padding-top: 20px;
  padding-bottom: 20px;
  font-size: large;
}

<body>
  <div>
    <div
      class="c0"
    >
      <h1>
        Home Page
      </h1>
      <div
        class="c1"
      >
        Zombie ipsum reversus ab viral inferno, nam rick grimes malum cerebro. De carne lumbering animata corpora quaeritis. Summus brains sit​​, morbo vel maleficia? De apocalypsi gorger omero undead survivor dictum mauris. Hi mindless mortuis soulless creaturas, imo evil stalking monstra adventus resi dentevil vultus comedat cerebella viventium. Qui animated corpse, cricket bat max brucks terribilem incessu zomby. The voodoo sacerdos flesh eater, suscitat mortuos comedere carnem virus. Zonbi tattered for solum oculi eorum defunctis go lum cerebro. Nescio brains an Undead zombies. Sicut malus putrid voodoo horror. Nigh tofth eliv ingdead.
        <br />
        <br />
        Cum horribilem walking dead resurgere de crazed sepulcris creaturis, zombie sicut de grave feeding iride et serpens. Pestilentia, shaun ofthe dead scythe animated corpses ipsa screams. Pestilentia est plague haec decaying ambulabat mortuos. Sicut zeder apathetic malus voodoo. Aenean a dolor plan et terror soulless vulnerum contagium accedunt, mortui iam vivam unlife. Qui tardius moveri, brid eof reanimator sed in magna copia sint terribiles undeath legionis. Alii missing oculis aliorum sicut serpere crabs nostram. Putridi braindead odores kill and infect, aere implent left four dead.
      </div>
    </div>
  </div>
</body>
`;
